### PR TITLE
add minutes handover interim board / board meeting

### PR DESCRIPTION
**Pull request by annefou**
- Check and to be merged when approved.

### CODE REVIEW

File: meetings/200325-board-meeting.md [hunk 1]
```diff
@@ -104,3 +104,128 @@
...
+Before starting the meeting, tips for using Zoom have been exchanged within the participants; for instance, on how to get the grid view (as we are few people).
+
+## Presentation of the new board and farewell/Thank you to the interim board
+
+Everyone gave a short presentation (interim board and then new board).
+
+
+# Introduction to Carpentry@UiO and Governance Documents
+
+- The interim board explained the rational behind these documents.
```
Comment (arockenberger) [inline meetings/200325-board-meeting.md]: rationale

File: meetings/200325-board-meeting.md [hunk 2]
```diff
@@ -104,3 +104,128 @@
...
+- the board secretary is in charge of the meeting minutes but does not have to write minutes all the time; Secretary is in charge of making sure there is a notetaker and that minutes are pushed to github.
+
+- The board believes there is a need for a Communication officer.
+
+- 2 members of the current board will be appointed for 1 year and 2 members for 2 years
+      --> how to do it?  
+      --> will be discussed later to decide how to do with the re-election of 2 members.
+
+The board agreed on the fact that the chair and vice-chair should have some experience. However, this will be re-discussed in a separate meeting.
+
```
Comment (arockenberger) [inline meetings/200325-board-meeting.md]: Should we add a link to the officer roles for the Carpentries Executive Council? https://github.com/carpentries/executive-council-info/blob/master/process/roles_responsibilities.md#officers

File: meetings/200325-board-meeting.md [hunk 3]
```diff
@@ -104,3 +104,128 @@
...
+
+## Dates for (remote) meetings in 2020
+
+The board think we should always allow remote meeting (even outside Covid-19 period).
+
+The board agreed to organize and lead 4 meetings per year. Next meeting will be decided at the end of a meeting (except for the first official board meeting where a doodle will be set-up.
+
```
Comment (arockenberger) [inline meetings/200325-board-meeting.md]: Did we say who sets up the Doodle? I can't remember…

Comment (arockenberger) [reply meetings/200325-board-meeting.md]: Aha, now I saw that it was me who was supposed to set up the Doodle…

File: meetings/200325-board-meeting.md [hunk 4]
```diff
@@ -104,3 +104,128 @@
...
+Mathworks (matlab) --> we all get this email. Should we offer matlab Carpentry workshop. Follow up with him.
+
+**Actions**: 
+- the new board will go through the governance documents and will update it accordingly.
+- The new board needs to check access to **uio-carpentry** github organization. Contact Annika if you needs to have write access to uio-carpentry github organization.
+- New board will organize appointment. Chair, vice-chair and communication officer;
+- Annika: doodle for setting up the next meeting.
+- Annika will write an on-boarding document for new board members.
+- Annika will follow up with Mathworks and see if they are willing to teach Carpentry matlab workshops at UiO.
```
Comment (arockenberger) [inline meetings/200325-board-meeting.md]: That was Elin's task, she has already done so, I will brief the Board the next time we meet about what they agreed on!

File: meetings/200325-board-meeting.md (hunk 3 above)
Comment (annefou) [reply meetings/200325-board-meeting.md]: I don't think we said who was supposed to set up the doodle. I assigned you. Happy to assign someone else if you want!

**Pull request merged**